### ISSUE TITLE
Liip Theme compatibility

### ISSUE THREAD

**Issue by nicodmf** [Spea/SpBowerBundle#91]
To begin thanks for the projet, this work easily and permit to integrate a real assets management.
## Explanation

For many projet, i use LiipTheme, This is very usefull to provide different interface for a same business case. So as spBower provide a simple tag for assetic : libname_(js|css), it is impossible to have a theme with jquery 1.8 and another with jquery 2.0.

Do you see a work-arround or a good solution.

I created an issue in LiipTheme too, maybe you will communicate to merge yours knowledges.

https://github.com/liip/LiipThemeBundle/issues/98

**Comment by dbu** [comment]
did this ever happen?

**Comment by Spea** [comment]
No, sadly not. I've currently not the time to maintain this bundle as I want, hopefully this will change in the next few weeks. If you still want/need this feature, I would really appreciate it, if you could create a PR for this.

**Comment by dbu** [comment]
i was just going through open issues on LiipThemeBundle, checking if we can clean up some of the things. if we end up needing it in the project i currently work on, i will create a PR, sure.
